### DISCUSSION TITLE
[solved] tuya ts0601 siren _TZE204_hcxvyxa5 - how to add support?

### DISCUSSION THREAD

**Discussion by astrakid** [Koenkk/zigbee2mqtt#24262] [Devices]
hi,
i got this device, which is not supported yet.

`modelZigbee:TS0601
type:Router
nwk:8857
manuf id:4417
manufacturer:_TZE204_hcxvyxa5
power:Mains (single phase)
app version:74
hard version:1
zcl version:3
stack version:0
date code:
interviewed:true
configured:false
endpoint:1
profile:260
input clusters:genGroups (4)
genScenes (5)
manuSpecificTuya (61184)
genBasic (0)
output clusters:genOta (25)
genTime (10)
endpoint:242
profile:41440
output clusters:greenPower (33)
`

is it possible to copy an existing converer?

since i am very busy i am not able to investigate as deep as i would like to - but i need the siren. maybe someone has q quick idea to find a quick solution.

regards,
andre

**Comment by astrakid** [comment]
i've found a quick solution already:
line 7127 in tuya.js
        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_k7mfgaen', '_TZE204_fncxk3ob', '_TZE204_hcxvyxa5']),

**Reply by akzg21** [reply]
Sorry to bring up an old issue, but I just got the same siren and have the same issue.

Could you kindly provide me with some detail on how to apply the solution you mentioned above?

Thanks

**Reply by astrakid** [reply]
finally i simply installed all the latest updates and it worked. before i changed line 7127 in tuya.js.

**Reply by akzg21** [reply]
I'll try complete all updates.
Where can I find tuya.js file?
